src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,20 @@
+package duke;
+
+import javafx.application.Application;
+
+/**
+ * Main class of Duke
+ *
+ * @author Jay Aljelo Saez Ting
+ */
+public class Duke {
+    /**
+     * Initialises and runs the Duke chatbot.
```

Comment:
Should this be "Initialize and runs.."? (or Initialise for British English)

Reply:
It is intended to be British English, though I guess this module prefers American spelling, so I could just change it. Thanks!